src/services/baseService.ts

```diff
@@ -193,6 +198,50 @@ export abstract class BaseService {
     return Utils.get(this.options, key, defaultValue);
   }
 
+  /**
+   * Spawn a Node child process with predefined environment variables
+   * @param command CLI Command - NO ARGS
+   * @param args Array of arguments for CLI command
+   * @param env Additional environment variables to be set in addition to
+   * predefined variables in `serverless.yml`
+   */
+  protected spawn(command: string, args?: string[], env?: any): Promise<void> {
```

Comment:
I don't think every service would need this.  I'd suggest we just move it into the offline service?

Reply:
I guess that's true... This would never be used in a CI/CD process, only by local devs. So it makes sense that it would only live in offline